cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -71,13 +71,21 @@ import (
 	"k8s.io/controller-manager/pkg/informerfactory"
 	"k8s.io/controller-manager/pkg/leadermigration"
 	"k8s.io/klog/v2"
+
+	"k8s.io/apimachinery/pkg/util/uuid"
+	kubefeatures "k8s.io/kubernetes/pkg/features"
+	"k8s.io/utils/clock"
```

Comment:
no extra block

Reply:
still open

Reply:
I've manually reverted this but my editor seems to be forcing it. Isn't it convention for the third block to be module/organization level imports which should include the entirety of k/k?